tools/esbuild/index.bzl

```diff
@@ -14,6 +14,12 @@ def esbuild(name, deps = [], mapping_targets = [], testonly = False, **kwargs):
     _esbuild(
         name = name,
         deps = devmode_targets,
+        # Inline source contents to make debugging easier. Contents are inlined by default
+        # in ESBuild but `@bazel/esbuild` sets the default to `false`. Inlining sources is
+        # helpful as otherwise developers would need to manually wire up the Bazel execroot
+        # as workspace in the Chrome devtools.
+        # https://github.com/bazelbuild/rules_nodejs/blob/c30a26c13d20dac48dc9f220370cb02a317b13f3/packages/esbuild/esbuild.bzl#L333.
+        sources_content = sources_content if sources_content else True,
```

Comment:
This will also result in `True` correct?

If we want to default it as `True`, we should just set the initial value to `True` and pass it in here.

Reply:
I followed this pattern because above in the signature it's not easy to incorporate such a large comment, but as discussed offline I'll look into making this a little more obvious. Maybe just moving it up..